django_mongodb/fields/auto.py

```diff
@@ -32,7 +32,7 @@ def get_prep_value(self, value):
             raise ValueError(f"Field '{self.name}' expected an ObjectId but got {value!r}.") from e
 
     def db_type(self, connection):
-        return "ObjectId"
+        return "objectId"
```

Comment:
why does this need to get lowercased?

Reply:
To match what MongoDB requires. See the table on the page linked in the PR description.